.eslintrc.js

```diff
@@ -1365,6 +1365,17 @@ module.exports = {
         'react/jsx-fragments': 'error',
       },
     },
+    {
+      files: ['x-pack/test/functional/apps/**/*.{js,ts}'],
```

Comment:
I think we will want to include the non x-pack test folder as well :)

Reply:
Yes :) I was just afraid to see hundreds of errors 🙈 Pushed in the next commit